dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestExceptionHandlerActor.java

```diff
@@ -0,0 +1,124 @@
+    }
+
+    private final ActorContext<DssRestExceptionHandlerCommand> context;
+    private final Map<Class<? extends DssRestActorService>, Map<Class<? extends Exception>, Method>> exceptionHandlerMap;
+
+    public DssRestExceptionHandlerActor(ActorContext<DssRestExceptionHandlerCommand> context, Map<Class<? extends DssRestActorService>, Map<Class<? extends Exception>, Method>> exceptionHandlerMap) {
+        this.context = context;
+        this.exceptionHandlerMap = exceptionHandlerMap;
+    }
+
+    public Behavior<DssRestExceptionHandlerCommand> dssRestExceptionHandlerActor() {
+        return Behaviors.receive(DssRestExceptionHandlerCommand.class)
+                .onMessage(DssRestExceptionHandlerCommandRequest.class, this::exceptionHandling)
+                .build();
+    }
+
+    private Behavior<DssRestExceptionHandlerCommand> exceptionHandling(DssRestExceptionHandlerCommandRequest commandRequest) {
+        Exception e = commandRequest.getException();
+        HttpResponseStatus httpResponseStatus;
+        DssRestServiceResponse dssRestServiceResponse;
+
+        if (e instanceof DssRestRequestMappingException) {
+            context.getLog().error("Json request mapping error: ", e);
+            httpResponseStatus = HttpResponseStatus.BAD_REQUEST;
+        } else {
+            context.getLog().error("Handling rest request error: ", e);
+            httpResponseStatus = HttpResponseStatus.INTERNAL_SERVER_ERROR;
+        }
+
+        dssRestServiceResponse = findExceptionHandleMethod(commandRequest);
+
+        replyRequest(commandRequest.getRequest(), httpResponseStatus, dssRestServiceResponse);
+
+        return Behaviors.same();
+    }
+
+    private DssRestServiceResponse findExceptionHandleMethod(DssRestExceptionHandlerCommandRequest commandRequest) {
+        Map<Class<? extends Exception>, Method> serviceExceptionHandlerMap = exceptionHandlerMap.get(commandRequest.getService().getClass());
+
+        if (containMethod(serviceExceptionHandlerMap, commandRequest)){
+            Method method = serviceExceptionHandlerMap.get(commandRequest.getException().getClass());
+            return invokeMethod(method, commandRequest);
+        }
+
+        Map<Class<? extends Exception>, Method> globalExceptionHandler = exceptionHandlerMap.get(DssRestActorService.class);
+        if (containMethod(globalExceptionHandler, commandRequest)){
+            Method method = globalExceptionHandler.get(commandRequest.getException().getClass());
+            return invokeMethod(method, commandRequest);
+        }
+
+        return null;
+    }
+
+    private DssRestServiceResponse invokeMethod(Method method, DssRestExceptionHandlerCommandRequest commandRequest) {
+        try {
+            Constructor<?> declaredConstructor = method.getDeclaringClass().getDeclaredConstructor();
+            declaredConstructor.setAccessible(true);
+
+            Object exceptionHandler = declaredConstructor.newInstance();
```

Comment:
Do i have to generate new exception handler instance every exception handling?

Reply:
I think it just need one. I didn't think about this. Thank you.

I try to code an instance of exceptionHandler to create once.